<div style="background-color: #e7f3fe; border-left: 6px solid #2196F3; padding: 10px; margin: 10px 0; color: #000;">
    <strong>Note:</strong> This is a custom blue text box. You can put any content here, including <em>formatted text</em> or even <a href="https://www.example.com">links</a>.
</div>

 # <strong> Phase 1: Library Installs and Imports <strong>
 ***

In [1]:
%pip install yfinance

     ---------------------------------------- 0.0/3.0 MB ? eta -:--:--
     ---------------------------------------- 3.0/3.0 MB 19.7 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
  Created wheel for peewee: filename=peewee-3.17.9-py3-none-any.whl size=139095 sha256=7f97b543b972fa401c43fcb9c6cc530920da10ff83a4bab6a62daf8831d8cc4a
  Stored in directory: c:\users\reyes\appdata\local\pip\cache\wheels\43\ef\2d\2c51d496bf084945ffdf838b4cc8767b8ba1cc20eb41588831
Successfully built peewee
Note: you may need to restart the kernel to use updated packages.


In [ ]:
import yfinance as yf
import pandas as pd

In [4]:
from Models.RY_Auto_LinearReg import LinearReg_model
from Models.RY_Auto_AuReg import AuReg_model
from Models.RY_Auto_MovAve import MovAve_model
from Models.RY_Auto_ARIMA import ARIMA_model
from Models.RY_Auto_SARIMA import SARIMA_model
from Models.RY_Auto_GARCH import GARCH_model

ImportError: cannot import name 'LinearReg_model' from 'Models.RY_Auto_LinearReg' (c:\Users\reyes\OneDrive\(0) Research and Coding\(2) Mapua\DS150 - Stock Predictions\Models\RY_Auto_LinearReg.py)

 # <strong> Phase 2: Stock Vars. Preprocessing
 ***

 ## _Phase 2.1: Tesla, Incorporated (NASDAQ: TSLA)_

 ## _Phase 2.2: GameStop Corporation (NYSE: GME)_

 ## _Phase 2.3: Advanced Micro Devices, Incorporated (NANSDAQ: AMD)_

 ## _Phase 2.4: Apple, Incorporated (NASDAQ: AAPL)_

 ## _Phase 2.5: SPDR S&P 500 ETF Trust (ARCX: SPY)_

 # <strong> Phase 3: Stock Vars. Prediction Modeling
 ***

 ## _Phase 3.1: Tesla, Incorporated (NASDAQ: TSLA)_

 ## _Phase 3.2: GameStop Corporation (NYSE: GME)_

 ## _Phase 3.3: Advanced Micro Devices, Incorporated (NANSDAQ: AMD)_

 ## _Phase 3.4: Apple, Incorporated (NASDAQ: AAPL)_

 ## _Phase 3.5: SPDR S&P 500 ETF Trust (ARCX: SPY)_

 # <strong> Phase 4: Findings and Descriptions
 ***